docs/source/release_notes/lts/next.rst

```diff
@@ -88,6 +88,118 @@ introduced by the next LTS version are:
   might still modify this to better follow their conventions (eg.
   for RPM the original locations are used)
 
+* The :mod:`avocado.core.utils.vmimage` library now allows users to
+  expand the builtin list of image providers.  If you have a local
+  cache of public images, or your own images, you can quickly and
+  easily register your own providers and thus use your images on your
+  tests.
+
+* Avocado can record the output generated from a test, which can then
+  be used to determine if the test passed or failed.  This feature is
+  commonly known as "output check".  Traditionally, users would choose
+  to record the output from ``STDOUT`` and/or ``STDERR`` into separate
+  streams, which would be saved into different files.  Some tests suites
+  actually put all content of ``STDOUT`` and ``STDERR`` together, and
+  unless we record them together, it'd be impossible to record them in
+  the right order.  This version introduces the ``combined`` option
+  to ``--output-check-record`` option, which does exactly that: it
+  records both ``STDOUT`` and ``STDERR`` into a single stream and
+  into a single file (named ``output`` in the test results, and
+  ``output.expected`` in the test data directory).
+
+* A new varianter plugin has been introduced, based on PICT.  PICT is
+  a "Pair Wise" combinatorial tool, that can generate optimal
+  combination of parameters to tests, so that (by default) at least a
+  unique pair of parameter values will be tested at once.
+
+* A new (optional) plugin is avaiable, the "result uploader".  It
+  allows job results to be copied over to a centralized results server
+  at the end of job execution.  Please refer to
+  :ref:`results-upload-plugin` for more information.
+
+* The ``default_parameters`` mechanism for setting default parameters
+  on tests has been removed.  This was introduced quite early in the
+  Avocado development, and allowed users to set a dictionary at the
+  class level with keys/values that would serve as default parameter
+  values.  The recommended approach now, is to just provide default
+  values when calling ``self.parameters.get`` within a test method,
+  such as ``self.parameters.get("key",
+  default="default_value_for_key")``.
+
+* The ``__getattr__`` interface for ``self.params`` has been removed.  It
+  used to allow users to use a syntax such as ``self.params.key`` when
+  attempting to access the value for key ``key``.  The supported syntax
+  is ``self.params.get("key")`` to achieve the same thing.
+
+* The :mod:`avocado.utils.vmimage` library now contains support for
+  Avocado's own JeOS image.  A nice addition given the fact that
```

Comment:
I think short description like `JeOS [Just Enough Operation System - Stripped-out Fedora with minimal deps]`. It's not long and can promote the concept. People don't know they need it :smile: